site/docs/uk/plugins/entity-parser.md

```diff
@@ -0,0 +1,238 @@
+
+- Якщо ви не вкажете `textSanitizer`, буде використано [`sanitizerHtml`](https://github.com/quadratz/telegram-entities-parser/blob/main/src/utils/sanitizer_html.ts) як типовий санітайзер.
+- Якщо вказати значення `false`, очищення буде пропущено, а виведений текст буде збережено як оригінал.
+  Це не рекомендується, оскільки може призвести до некоректного рендерингу і зробити вашу програму вразливою до XSS-атак.
+  Якщо ви вибрали цю опцію, забезпечте належну обробку результату.
+- Якщо ви надасте функцію, вона буде використовуватися замість типового санітайзера.
+
+```ts
+const myTextSanitizer: TextSanitizer = (options: TextSanitizerOption): string =>
+  // Замінюємо небезпечні символи.
+  options.text.replaceAll(/[&<>"']/, (match) => {
+    switch (match) {
+      case "&":
+        return "&amp;";
+      case "<":
+        return "&lt;";
+      case ">":
+        return "&gt;";
+      case '"':
+        return "&quot;";
+      case "'":
+        return "&#x27;";
+      default:
+        return match;
+    }
+  });
+
+// Застосовуємо санітайзер.
+const entitiesParser = new EntitiesParser({ textSanitizer: myTextSanitizer });
+```
+
+## Випадки, коли краще не використовувати цей пакет
+
+Якщо ви зіткнулися з проблемами, подібними до наведених нижче, можливо, вам вдасться вирішити їх без використання цього пакету.
+
+### Копіювання та пересилання одного і того ж повідомлення
+
+Використовуйте [`forwardMessage`](https://core.telegram.org/bots/api#forwardmessage) для пересилання повідомлень будь-якого типу.
+
+Ви також можете використовувати [`copyMessage`](https://core.telegram.org/bots/api#copymessage), який виконає ту ж саму дію, але не міститиме посилання на оригінальне повідомлення.
+[`copyMessage`](https://core.telegram.org/bots/api#copymessage) поводиться так, ніби копіює повідомлення і надсилає його назад у Telegram, при цьому воно відображається як звичайне повідомлення, а не як переслане.
+
+```ts
+bot.on(":text", async (ctx) => {
+  // Ідентифікатор цільового чату для надсилання.
+  const chatId = -946659600;
+  // Пересилаємо поточне повідомлення без посилання на початкове повідомлення.
+  await ctx.copyMessage(chatId);
+  // Пересилаємо поточне повідомлення з посиланням на початкове повідомлення.
+  await ctx.forwardMessage(chatId);
+});
+```
+
+### Відповідь на повідомлення зі зміненим форматуванням тексту
+
+Ви можете легко відповідати на вхідні повідомлення за допомогою HTML, Markdown або сутностей.
+
+```ts
+bot.on(":text", async (ctx) => {
+  // Відповідаємо за допомогою HTML.
+  await ctx.reply("<b>жирний</b> <i>курсив</i>", { parse_mode: "HTML" });
+  // Відповідаємо за допомогою Telegram Markdown V2.
+  await ctx.reply("*жирний* _курсив_", { parse_mode: "MarkdownV2" });
+  // Відповідаємо за допомогою сутностей.
+  await ctx.reply("жирний курсив", {
+    entities: [
+      { offset: 0, length: 5, type: "bold" },
+      { offset: 5, length: 6, type: "italic" },
+    ],
+  });
```

Comment:
```suggestion
  await ctx.reply("жирний курсив", {
    entities: [
      { offset: 0, length: 6, type: "bold" },
      { offset: 7, length: 6, type: "italic" },
    ],
  });
```

Reply:
@niusia-ua, we should accept this suggestion.

Our code result:
![image](https://github.com/user-attachments/assets/8b245a52-9d0e-4d67-ac5c-78217a5f9692)

Suggested code result:
![image](https://github.com/user-attachments/assets/96499031-43ff-4993-a962-dc4e38630cbd)